Tests/AuthServiceTests/Validators/CreateUserRequestValidatorTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿using AuthenticationService.Utils;
+using DTO.RestRequests;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AuthenticationServiceTests.Validators
```

Comment:
In Authentification. CreateUserRequestTests. Are you seriously?

Reply:
Also, we already have sth like that in  user service" createuserrequestvalidator